doc/contributing/contributing-to-documentation.chapter.md

```diff
@@ -120,3 +120,34 @@ Additional syntax extensions are available, though not all extensions can be use
   > :   green fruit with red flesh
 
 For contributing to the legacy parts, please see [DocBook: The Definitive Guide](https://tdg.docbook.org/) or the [DocBook rocks! primer](https://web.archive.org/web/20200816233747/https://docbook.rocks/).
+
+## Diagrams
+
+Diagrams are provided to contextualize the amount of information and processes
+into something more visual.
```

Comment:
I don't think we need to motivate this here.

```suggestion
```

But we may add a note at the end that more diagrams are welcome.